GNUmakefile

```diff
@@ -0,0 +1,25 @@
+TEST?="./provider"
```

Comment:
The [module was renamed](https://github.com/claranet/terraform-provider-zabbix/pull/7/commits/2536904ea4293210e24914f06527ae81f513458e) :p 
```suggestion
TEST?="./zabbix"
```

Reply:
good catch thanks !